python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -388,6 +399,43 @@ def terminate_scheduled_nodes(self):
         self.nodes_to_terminate = []
         self.update_worker_list()
 
+    def drain_nodes_via_gcs(self):
```

Comment:
can we make this function selfless ?

Reply:
Good idea.

Reply:
Well, not selfless but now takes a list of node_ids as argument to match self.provider.terminate_nodes